loopchain/utils/command_arguments/__init__.py

```diff
@@ -65,7 +67,7 @@ def command(self, value):
 
 attributes = {
     Type.ServiceType:
-        Attribute("service_type", type=str,
+        Attribute("service_type", type=str, default='citizen', nargs=OPTIONAL,
```

Comment:
Why don't you use `'?'` instead of `OPTIONAL` constant?
No more usage `OPTIONAL` in this scope and if you need more `nargs` in future, it would be better to provide various options(`?, +, *, etc...`)

Reply:
OK. I thought it might be easier to recognize if it's set with specific meaning.